docs/CONFIGURATION.md

```diff
@@ -252,10 +252,21 @@ The agent table configures Telegraf and the defaults used across all plugins.
   or "gosmi" which translates using the built-in gosmi library.
 
 - **ignore_error_inputs**:
-  If set to true, discard the input plugins that produce the error during initialization.
-  If set to false, the program will exit when an input plugin has an error occurred during the initialization.
+  Together with plugin.ignore_init_error it forms a configuration: 
+
+  If (plugin.ignore_init_error || ignore_error_inputs) is true, discard the input plugins that produce the error during initialization.
+
+  If (plugin.ignore_init_error || ignore_error_inputs) is false, the program will exit when an input plugin has an error occurred during the initialization.
+
   Default: false
 
+  | input.ignore_init_error | agent.ignore_error_inputs |   Is Ignore    |
+  |:-----------------------:|:-------------------------:|:--------------:|
+  |          true           |            true           |     true       |
+  |          true           |            false          |     true       |
+  |          false          |            true           |     true       |
+  |      false(default)     |        false(default)     | false(default) |
+
```

Comment:
```suggestion
  If (plugin.ignore_init_error || ignore_error_inputs) is true, discard the input plugins that produce the error during initialization. Otherwise, the program will exit when an input plugin has an error occurred during the initialization.

  Default: false
```